kong/plugins/basic-auth/access.lua

```diff
@@ -101,7 +105,7 @@ function _M.execute(conf)
   end
 
   if not credential or not validate_credentials(credential, given_password) then
-    return responses.send_HTTP_FORBIDDEN("Invalid authentication credentials")
+    return false, {status = 403, message = "Invalid authentication credentials"}
```

Comment:
why not just return `success`, `statuscode`, `message` without creating the intermediate table? More efficient and less GC.

same for the other plugins